person.rb

```diff
@@ -1,20 +1,30 @@
 require_relative 'nameable'
+require_relative 'book'
+require_relative 'rental'
+
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Just as the reviewer suggested, `rental` needs to have a `getter` too. Currently, it only has a `setter`. I'll suggest you update this. :+1: :wave: 

Here is a possible solution: 
```ruby
attr_accessor :name, :age, :rentals
```